src/Stryker.Core/Stryker.Core/MutationTest/MutationTestProcess.cs

```diff
@@ -94,10 +94,15 @@ public void Mutate()
             _reporter.OnMutantsCreated(_input.ProjectInfo.ProjectContents);
         }
 
-        public void Test()
+        public void Test(int maxConcurrentTestRunners)
         {
-            var logicalProsessorCount = Environment.ProcessorCount;
-            var usableProcessorCount = logicalProsessorCount / 2;
+            var logicalProcessorCount = Environment.ProcessorCount;
+            if(maxConcurrentTestRunners <= logicalProcessorCount)
+            {
+                logicalProcessorCount = maxConcurrentTestRunners;
+            }
+            
+            var usableProcessorCount = Math.Max(logicalProcessorCount / 2, 1);
```

Comment:
the maxConcurrentTestRunners will never be hit this way. Take this example with maxConcurrentTestRunners = 4:
``` csharp
var logicalProcessorCount = Environment.ProcessorCount; // 8
if(maxConcurrentTestRunners <= logicalProcessorCount)
{
    logicalProcessorCount = maxConcurrentTestRunners; // 4
}
var usableProcessorCount = Math.Max(logicalProcessorCount / 2, 1); // 2
```

The maximum usableProcessorCount is half of the max.

Reply:
Ah yes, will fix that.